.github/workflows/tests_without_binary.yml

```diff
@@ -103,6 +103,8 @@ jobs:
           SKIP_COMPILATION=True PL_BACKEND="lightning_qubit" python -m pip install . -vv
 
       - name: Install backend device
+        # FIXME: Remove this condition with v0.37.0 release
+        if: matrix.pl_backend != 'lightning_tensor' && inputs.lightning-version != 'stable'
```

Comment:
This will run only for not `lightning_tensor` (all other devices) and never for stable, right?

This means this will never run, for:

-  any `stable` backend variant 
-  any of `lightning_tensor` version variants.

Reply:
This will be still triggered for 1) `lightning_tensor` and `latest`, and 2) any other device and stable/latest.